sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1426,6 +1434,28 @@ static inline void _event_post(gnrc_netif_t *netif)
 #endif
 }
 
+static void _process_receive_stats(gnrc_netif_t *netdev, gnrc_pktsnip_t *pkt)
+{
+#ifdef MODULE_NETSTATS_NEIGHBOR
+    gnrc_netif_hdr_t *hdr;
+    const uint8_t *src = NULL;
+    gnrc_pktsnip_t *netif = gnrc_pktsnip_search_type(pkt, GNRC_NETTYPE_NETIF);
+
+    if (netif == NULL) {
+        return;
+    }
+
+    size_t src_len;
+    hdr = netif->data;
+    src = gnrc_netif_hdr_get_src_addr(hdr);
+    src_len = hdr->src_l2addr_len;
+    netstats_nb_update_rx(&netdev->netif, src, src_len, hdr->rssi, hdr->lqi);
+#else
+    (void) netdev;
+    (void) pkt;
+#endif
```

Comment:
Shouldn't

```suggestion
if (IS_USED(MODULE_NETSTATS_NEIGHBOR) {
    gnrc_netif_hdr_t *hdr;
    const uint8_t *src = NULL;
    gnrc_pktsnip_t *netif = gnrc_pktsnip_search_type(pkt, GNRC_NETTYPE_NETIF);

    if (netif == NULL) {
        return;
    }

    size_t src_len;
    hdr = netif->data;
    src = gnrc_netif_hdr_get_src_addr(hdr);
    src_len = hdr->src_l2addr_len;
    netstats_nb_update_rx(&netdev->netif, src, src_len, hdr->rssi, hdr->lqi);
}
```

Also work here (also if suggestion is taken, please fix indentation ;-))